scripts/module/keybinder.nut

```diff
@@ -39,26 +43,26 @@ function partition( input, separator ) {
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
   if (cmd == "bind") {
-    text = text.tolower();
-    local bindTable = {};
-    if (text.len() == 0) {
+    if (!text) {
       local currbinds = "Current binds: \n";
       if (binds[player.ID]) {
         foreach (key, bind in binds[player.ID]) {
-          currbinds += key + ": " + bind + "\n";
+          currbinds += "'" + key + "'" + ": '" + bind + "'\n";
         }
       }
       MessagePlayer( currbinds + "Usage:\n" +
-                    "Bind key: /" + cmd + " <key> <command>\n" +
-                    "Unbind:   /" + cmd + " <key>\n" +
-                    "Get help: /" + cmd, player);
+                     "Bind key: /" + cmd + " <key> <command>\n" +
+                     "Unbind:   /" + cmd + " <key>\n" +
+                     "Get help: /" + cmd, player);
       return;
     }
+    text = text.tolower();
+    local bindTable = {};
     local argArray = partition(text, " ");
     local key = argArray[0];
-     if (key in keys) {
-      key = keys[key];
-    } else if (isNum(key)) {
+    if (key in keys) {
+      key = keys.rawget(key);
```

Comment:
Why all the `raw*` stuff instead of `[]`, `<-` etc?

Reply:
I found it somewhere and had a brainfart.
All uses except the `.rawdelete()` should be fixed now. :+1:

Reply:
`.rawdelete()` is kept because there's no other way of doing the same thing, `x[y]<- null` cleans out the slot but doesn't delete what was in there - it might still be floating in memory somewhere.